crypto/asn1/d2i_pu.c

```diff
@@ -32,7 +32,7 @@ EVP_PKEY *d2i_PublicKey(int type, EVP_PKEY **a, const unsigned char **pp,
     } else
```

Comment:
A very trivial 'near-by' coding style nit - needs braces...

Reply:
Yeah, I didn't touch that.